docs/reference/contributing/guidelines/workflow.md

```diff
@@ -52,7 +52,8 @@ Pull requests on GitHub have to meet the following requirements to keep the code
 - All new features and enhancements require documentation, tests and user guides for us to accept them. Please link each pull request to all relevant documentation and test pull requests.
 - Avoid merging commmits. (Always rebase when possible.)
 - Comment in the pull request on every change (rebase or new commits). This helps reviewers to be up to date with changes
-- Pull requests should fix a bug, add a feature or refactor.
+- Pull requests should fix a bug, add a feature or refactor. 
+  The only exception are third-party version updates (for instance MbedTLS or Nanostack releases for Mbed OS). These updates should provide release notes for updated version with documentation (a reference to external changelog or release notes).
```

Comment:
Respect the trademark.

```diff
- MbedTLS
+ Mbed TLS
```

Reply:
We don't always have a reference to an external changelog or release notes, depending on the library being updated and if the release is a pre-release version.

```diff
- These updates should provide release notes for updated version with documentation (a reference to external changelog or release notes).
+ These updates should provide Mbed OS release notes in the pull request description, or link to an external changelog or release notes.
```